sections/sections-app/src/webapp/options.jsp

```diff
@@ -52,12 +61,18 @@
         </t:div>
     
         <t:div styleClass="act verticalPadding">
-            <h:commandButton
-                action="#{optionsBean.update}"
+            <h:commandButton                
                 value="#{msgs.update}"
                 styleClass="active"
-                rendered="#{optionsBean.sectionOptionsManagementEnabled}"
-                disabled="#{optionsBean.confirmMode}" />
+                rendered="#{optionsBean.sectionOptionsManagementEnabled and optionsBean.management == 'internal'}"
+                disabled="#{optionsBean.confirmMode}"
+                onclick="return update('internal');" />
+            <h:commandButton                
+                value="#{msgs.update}"
+                styleClass="active"
+                rendered="#{optionsBean.sectionOptionsManagementEnabled and optionsBean.management == 'external'}"
+                disabled="#{optionsBean.confirmMode}"
+                onclick="return update('external');" />
```

Comment:
You should use jQuery to register an event handler for the button see http://api.jquery.com/on/
The reason for doing this to eliminate the pollution of Javascript in HTML. In the end we want simple concise html with no CSS or Javascript pollution.

I know it seems like were nit picking but at some point we have to get handle on the mess here, this is one of the reasons that make Sakai so difficult to move forward because of all the bad practices.

Reply:
Hi @ern 

This has been discussed in the S2U meeting, it has been done by UPV and has been tested by the S2U, they put a lot of effort into testing. As you can see this has a long history of comments in the Jira, unfortunately, they can't improve this right now and I'm afraid this gets lost. Would you mind to merge it as is?

Thank you so much.